### ISSUE TITLE
Dependabot auto merge updates

### ISSUE THREAD

**Issue by EmaSuriano** [jsonresume/resume-cli#509]
Hello,

I'm one of the members of the json-resume team. I am getting every week notifications regarding the Dependabot updates, and I was wondering if we can enable the auto merge option inside the config. It's inside the [UI interface](https://app.dependabot.com/).

![image](https://user-images.githubusercontent.com/3399429/108825435-4b758e80-75c3-11eb-978b-ddcad0fe1cee.png)

Therefore, if we have the proper pr checkers, we don't have to worry about manually merging the automatic updates. A very clear example would be this pr --> https://github.com/jsonresume/resume-cli/pull/487 Which is already covered by the running the `lint` command at pr time :)

**Comment by thomasdavis** [comment]
Yeah, it is kind of annoying, why don't we trial it for a bit, do you have enough permissions to enable it?

**Comment by EmaSuriano** [comment]
Yeah I think have all the permissions. The bad news is that Dependabot decided to remove this feature (https://github.com/dependabot/dependabot-core/issues/1973#issuecomment-640918321), so we cannot use only Dependabot to handle this. But the community came up with a Github Action to make this happen: https://github.com/marketplace/actions/dependabot-auto-merge

Before adding it, I would love to discuss the idea of adding a new action into the project and also, we have to provide one Github token and given this is a shared project I don't know how we want to handle this :)
